src/main/java/spring/boot/bookstore/service/emailsender/EmailService.java

```diff
@@ -0,0 +1,85 @@
+package spring.boot.bookstore.service.emailsender;
+
+import java.util.Properties;
+import javax.activation.DataHandler;
+import javax.activation.DataSource;
+import javax.activation.FileDataSource;
+import javax.mail.Authenticator;
+import javax.mail.BodyPart;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Multipart;
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeBodyPart;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.MimeMultipart;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import spring.boot.bookstore.exception.MessageSenderException;
+import spring.boot.bookstore.model.Order;
+
+@Service
+public class EmailService {
```

Comment:
please add the interface
```suggestion
public class EmailServiceImpl implements EmailService {
```